internal/toc/toc.go

```diff
@@ -106,20 +107,21 @@ func saveToc(rootPath string, tocBytes []byte) error {
 }
 
 func insert(table model.Toc, ctm model.CatalogThingModel) error {
-	// TODO: if author and manufacturer are equal, it is offical, right?
-	tmid, err := model.ParseTMID(ctm.ID, false)
+	official := internal.Prep(ctm.Manufacturer.Name) == internal.Prep(ctm.Author.Name)
```

Comment:
Wouild it make sense to consolidate these checks in one place?

Reply:
Yes, which place do you propose? I suggested the ThingModel struct? Something like `func (tm *ThingModel) isOfficial()`?

Reply:
I don't think that'd work for `TocThing`

Reply:
Correct, I forgot that I removed the struct embedding. The `TocThing` relies on the `ThingModel`. The latter is used for parsing the fields from the TM, the latter is just a restructuring of the fields. How about we put the `isOfficial()` method with the `ThingModel` and save it's outcome into a plain field in the `TocThing`? Not perfect, yet, but maybe better than code duplication?

Reply:
I will make a note and merge for now.